src/buttons/Button.js

```diff
@@ -181,7 +207,7 @@ Button.propTypes = {
   fontFamily: PropTypes.string,
 };
 
-const styles = StyleSheet.create({
+const stylesObject = {
```

Comment:
@filiphosko Totally overlooked the description 😄 I will take a look at in more detail and let u know